### ISSUE TITLE
Running evaluation code without training

### ISSUE THREAD

**Issue by AtiqEmenent** [vb000/NeuriCam#8]
I am trying to perform evaluation task for your code without doing the trainig process for myself. I am using pre-trained model ([pretrained.pth.tar]) & spynet weights available in the Evaluation section of this GitHub page. But I am not getting results (for evaluation) as described in the paper. I have genereated LR images using cv2.resize() with cv2.INTER_CUBIC as the interpolation parameter (image alreday converted to LAB color scheme using cv2.COLOR_BGR2LAB ). Should i have to re-train the model myself or should i generate LR frames using MatlaB resize(as told in the paper) or I am doing some other mistake. Please guide.

My results example: For the 'walk' video frames of the dataset Vid4 i am getting PSNR-RGB of about 21.08.

Evalution command i used is : !python evaluate.py --lr_dir=lr-set-lab --key_dir=key-set --target_dir=hr-set  --output_dir=sr-set  --model_dir=experiments/bix4_keyvsrc_attn --restore_file=pretrained --file_fmt="frame%d.png"

**Comment by vb000** [comment]
Hi @AtiqEmenent, Matlab resize performs anti-aliasing along with the interpolation, which gives very different results compared to cv2.resize. This was something I stumbled upon during the development as well. But most of the prior works use Matlab's function, so I used the same for easier comparison. Here's a reference Matlab [function]( https://github.com/xinntao/EDVR/blob/master/scripts/matlab_scripts/generate_bicubic_img.m) for bicubic downsampling. Also make sure normalization is similar to what the pretrained model uses [here](https://github.com/vb000/NeuriCam/blob/bc8510da9ea227df7c7a0f9658b67731b63242b0/utils.py#L113) during the RGB to lab conversion.

**Comment by Justarrrrr** [comment]
hi, I want to know if all input images need to be converted to the LAB color space.
